### PR TITLE
Multiple fixes for Root on ZFS guide

### LINKED ISSUE #22: Fedora ROOT ON ZFS Guide
It would be nice if we could have a Fedora Root On Zfs Guide on the wiki

### System information

Type | Version/Name
 --- | --- 
Distribution Name	| Fedora
Distribution Version	| 32
Linux Kernel	| 
Architecture	| 
ZFS Version	| 
SPL Version	|

### PR DESCRIPTION
**Pull request by ne9z**
Fixed a couple of minor issues. Test run completed.

@gmelikov 

Closes #190
Closes #22

### CODE REVIEW

File: docs/Getting Started/Fedora/Root on ZFS/1-preparation.rst [hunk 1]
```diff
@@ -6,10 +6,13 @@ Preparation
 .. contents:: Table of Contents
    :local:
 
+#. Disable Secure Boot. ZFS modules can not be loaded of Secure Boot is enabled.
```
Comment (gmelikov) [inline docs/Getting Started/Fedora/Root on ZFS/1-preparation.rst:9]:
```suggestion
#. Disable Secure Boot. ZFS modules can not be loaded if Secure Boot is enabled.
```

File: docs/Getting Started/RHEL-based distro/RHEL 8-based distro Root on ZFS/1-preparation.rst [hunk 2]
```diff
@@ -6,10 +6,10 @@ Preparation
 .. contents:: Table of Contents
    :local:
 
+#. Disable Secure Boot. ZFS modules can not be loaded of Secure Boot is enabled.
```
Comment (gmelikov) [inline docs/Getting Started/RHEL-based distro/RHEL 8-based distro Root on ZFS/1-preparation.rst:9]:
```suggestion
#. Disable Secure Boot. ZFS modules can not be loaded if Secure Boot is enabled.
```

Comment (ne9z) [reply docs/Getting Started/RHEL-based distro/RHEL 8-based distro Root on ZFS/1-preparation.rst]: Thanks, fixed.

**Comment by ne9z** [comment]
There seems to be a issue with the pipeline:
```
ERROR: Cannot find command 'git' - do you have 'git' installed and in your PATH?
```

**Comment by gmelikov** [comment]
Yes, I'll look at it in a week.20.08.2021, 13:43, "ne9z" ***@***.***>:
There seems to be a issue with the pipeline:
ERROR: Cannot find command 'git' - do you have 'git' installed and in your PATH?

**Comment by ne9z** [comment]
I looked into the error, this is caused by the recent Debian Bullseye release.

Currently the site is built with ammaraskar/sphinx-action, which is using "buster" release. The release of bullseye causes `apt update` to fail, and therefore `apt install git` also fails. With git missing, the source repo can not be cloned.

One solution would require us to drop ammaraskar/sphinx-action altogether and be content with `make html`. This has the drawback of removing warnings for format errors.

The other would be convert gmelikov/sphinx-notfound-page.git to an http download.

**Comment by gmelikov** [comment]
@ne9z I've fixed CI in https://github.com/openzfs/openzfs-docs/pull/194 .

Please rebase and remove CI changes.

And just wanted to tell - let's discuss global changes before any PRs to save your time (CI-related).

**Comment by ne9z** [comment]
Sorry for the CI change - reverted.

**Comment by ne9z** [comment]
I think it's ready for merge. @gmelikov

**Comment by gmelikov** [comment]
@ne9z if you're ok with squashing in one commit - nevermind, otherwise please prepare commits with needed granularity and appropriate meaningful description.

either way, thank you!

**Pull request merged**